### DISCUSSION TITLE
Looking for improved documentation for DGSwap

### DISCUSSION THREAD

**Discussion by zxstim** [kaiachain/kaia-dapp-mono#193] [Requests]
Go through DGSwap:
- GitHub repo (https://github.com/dragon-swap-klaytn)
- Docs (https://docs.dgswap.io/)

and suggest improvements

**Comment by PaulElisha** [comment]
what would you expect to be covered in the improvement?

**Reply by zxstim** [reply]
prob priority is how can other developers interact with dgswap contracts. so more frontends can be built, helper tools, etc.

**Reply by PaulElisha** [reply]
noted, for both v2 and v3?

**Reply by zxstim** [reply]
i think v2 has zero liquidity so v3 only is ok

**Reply by PaulElisha** [reply]
check out my first deliverable of an improved documentation and helper library for dgswap:

https://github.com/PaulElisha/dragonswap-docs

still working on improving it, sdk for swap and helpers, will be updated for the second deliverable.
